### PR TITLE
Add data sources tab

### PR DESCRIPTION
**Pull request by LizBaker**
Adds a data source tab and page with empty state
Closes https://github.com/newrelic/developer-website/issues/1511

## Screenshot(s)
Screen Shot 2021-08-17 at 3 07 13 PM

### CODE REVIEW

File: src/templates/ObservabilityPackDetails.js [hunk 1]
```diff
@@ -226,6 +226,7 @@ const ObservabilityPackDetails = ({ data, location }) => {
             <Tabs.BarItem id="alerts" count={pack.alerts?.length ?? 0}>
               Alerts
             </Tabs.BarItem>
+            <Tabs.BarItem id="data-sources">Data sources</Tabs.BarItem>
```
Comment (aswanson-nr) [inline src/templates/ObservabilityPackDetails.js:229]: I think we want to include the count of data sources in this tab title, similar to the other tabs. Worth mentioning that this is the combination of the `instrumentation` and `documentation` fields

**Review by aswanson-nr — CHANGES_REQUESTED** [review]

File: src/templates/ObservabilityPackDetails.js (hunk 1 above)
Comment (LizBaker) [reply src/templates/ObservabilityPackDetails.js:229]: I was wondering about this, it felt kind of weird to count those as individual items but i'm happy to add if that makes sense to you

Comment (rudouglas) [reply src/templates/ObservabilityPackDetails.js]: Ye I agree just to keep it consistent with the design of the other tabs (though also think weird)

File: src/templates/ObservabilityPackDetails.js [hunk 2]
```diff
@@ -226,7 +226,15 @@ const ObservabilityPackDetails = ({ data, location }) => {
             <Tabs.BarItem id="alerts" count={pack.alerts?.length ?? 0}>
               Alerts
             </Tabs.BarItem>
-            <Tabs.BarItem id="data-sources">Data sources</Tabs.BarItem>
+            <Tabs.BarItem
+              id="data-sources"
+              count={
+                (pack.instrumentation?.length ?? 0) +
+                (pack.documentation ? 1 : 0)
```
Comment (aswanson-nr) [inline src/templates/ObservabilityPackDetails.js:233]: Sorry to toss another one on here! But the documentation field will also be an array

Comment (LizBaker) [reply src/templates/ObservabilityPackDetails.js:233]: thank you 😂 I had it in my head that was a string

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**